src/material/chips/chips.md

```diff
@@ -1 +1,67 @@
-<!-- TODO: this guide and the related live examples need to be rewritten to reflect the new chips API. -->
+The chips allows users to view and interact with a list of information.
+
+<!-- example(chips-overview) -->
+
+### Interaction Patterns
+
+The chips components support 3 different user interaction patterns, each with their own container and chip elements:
+
+<mat-chip-listbox> and <mat-chip-option> : These elements implement a listbox accessibility pattern and should be used when the chips are presenting a set of user selectable options.
```

Comment:
I would add subheadings: #### Listbox, #### Grid, #### Custom

Reply:
done

Reply:
I named them Listbox, Text Entry and static List